src/CMakeLists.txt

```diff
@@ -28,6 +28,9 @@ set(projectPrefix MP_UNITS_)
 option(${projectPrefix}AS_SYSTEM_HEADERS "Exports library as system headers" OFF)
 message(STATUS "${projectPrefix}AS_SYSTEM_HEADERS: ${${projectPrefix}AS_SYSTEM_HEADERS}")
 
+option(${projectPrefix}BUILD_CXX_MODULES "Add C++ modules to the list of default targets" OFF)
```

Comment:
The same comment from docs applies here. It is not obvious what it does. Maybe something like "Enables C++ modules-based build" would be better.

Reply:
After a longer thought I think that this comment here is OK. The end user does not care how we build our tests and examples.
However, in the documentation, we can add a second sentence describing the outcome of this option for the local build.